README.md

```diff
@@ -154,6 +154,31 @@ Then more than 10 posts have five comments
 Then less than 200 posts have five comments
 ```
 
+#### Errors
+
+If the `error_key` environment variable is set, if that key is used as the initial step it will ignore any `data_key`
+setting.
+
+Example: `error_key=error`, `data_key=data`
+
+```gherkin
+Then the response has a list of posts       | {"data":[{"id": 12}]}
+
+Then the response has one error             | {"errors":[{"message": "Error"}]}
+
+Then the response has an error              | {"error": {"message": "Error}}
```

Comment:
`Has one error` works but `Has an error` does not. I modified `errors.feature` to use that wording and the test failed.

```shell
[burhan@callisto cucumber-rest-bdd (errors)] {1} $ docker-compose run --rm runner cucumber --fail-fast --tags ~@broken features/errors.feature
Feature: Handling error responses

  Background:           # features/errors.feature:3
    Given I am a client # lib/cucumber-rest-bdd/steps/resource.rb:10

  Scenario: Item not found returns an error               # features/errors.feature:6
    When I request the post "not found"                   # lib/cucumber-rest-bdd/steps/resource.rb:18
    Then the request fails because the post was not found # lib/cucumber-rest-bdd/steps/status.rb:45
    And the response contains at least one errors         # lib/cucumber-rest-bdd/steps/response.rb:23
    And the response has an error                         # lib/cucumber-rest-bdd/steps/response.rb:23
      Could not find a match for: has an error
       (RuntimeError)
      features/errors.feature:10:in `And the response has an error'
    And the response has one error with the attributes:   # lib/cucumber-rest-bdd/steps/response.rb:11
      | attribute | type   | value     |
      | Message   | string | Not Found |

Failing Scenarios:
cucumber features/errors.feature:6 # Scenario: Item not found returns an error

1 scenario (1 failed)
6 steps (1 failed, 1 skipped, 4 passed)
```